### ISSUE TITLE
[Feature Request] Please add subscene

### ISSUE THREAD

**Issue by dmarcelino** [Diaoul/subliminal#431]
Hi,

As previously mentioned in [#69](https://github.com/Diaoul/subliminal/issues/69#issuecomment-10256111)'s and [#78](https://github.com/Diaoul/subliminal/issues/78#issuecomment-5423237)'s comments, can you please add support for subscene?

I'm not sure if this helps but currently the Kodi addon for downloading subscene subs works quite well and the code is here:
https://github.com/manacker/service.subtitles.subscene

Thanks!

**Comment by paericksen** [comment]
+1 for subscene

**Comment by yagyeshgod** [comment]
Yes adding Subscene wil make this script complete. Waiting badly for it

**Comment by AbhiDGamer** [comment]
Please add subscene support

**Comment by bitzorro** [comment]
+1 love subscene fast and coreckt subbs.

**Comment by luigino** [comment]
Seconded.
There's already a subscene-api on PyPI.

**Comment by aldanor** [comment]
Thirded.

**Comment by euphoria360** [comment]
+1
I would love this too.
almost all of subtitles in my language is in subscene. so it would be a great feature.
any update on this?

**Comment by euphoria360** [comment]
maybe we could make dev more interested via this bountyscore thing!?
 what do you guys think?

**Comment by bitzorro** [comment]
+1👌

**Comment by iioodii** [comment]
++++1

**Comment by gauravjos** [comment]
+++++1

**Comment by aboodi0** [comment]
+1

**Comment by ZataH** [comment]
Any status on this?

**Comment by luigino** [comment]
Well, it's all volunteer work. If someone else can contribute perhaps they should.

OTOH it would be interesting to hear from Diaoul if he considers this viable, theoretically.

**Comment by Diaoul** [comment]
Well if someone puts work in a PR I'd be glad to review it as I don't have the resource atm to do it myself.

**Comment by euphoria360** [comment]
I'm not a python programmer, but know some very simple basics.

I found this in github and cloned it on my sysytem, seems it's functional. can this be integrated into subliminal?
https://github.com/mamins1376/Subscene-API

**Comment by luigino** [comment]
@euphoria360 
I know, he recently updated it.
I contacted the author a few days ago and he said he's going to work on a provider for subliminal.

So there's hope.

**Comment by Eisa-AlAwadhi** [comment]
+1 for subscene!

**Comment by vulcanjedi** [comment]
same here most the languages we want are in this provider please.

**Comment by dirkr07** [comment]
would love for this one to be added!

**Comment by MrDemous** [comment]
+1

**Comment by MrDemous** [comment]
Hey, this guy @mamins1376 made these commits to get subscene working on his subliminal fork. Will these commits help get subscene implemented?

https://github.com/mamins1376/subliminal/commits/develop

P.S. I'm not a developer so I have no idea what I'm talking about but, I just thought I should give you guys a heads-up in case you didn't see this before.

**Comment by Derkades** [comment]
Any updates? It looks like the work has already been done

**Comment by mamins1376** [comment]
Blocked on #950

**Comment by p0psicles** [comment]
Not blocked any more?

**Comment by dirkr07** [comment]
any update on this one?

**Comment by euphoria360** [comment]
@Diaoul @mamins1376 Any update on this? #950 seems fixed. Damn, I miss Subscene on Subliminal sooooo much.

**Comment by dausruddin** [comment]
Subscene can be a PITA thing to work with. Even if we use it as a regular user, the website doesn't work well. I wonder why the dev doesn't optimize and design the website properly when they generate a lot of money.

But sadly languages that people around me and I understand, are only being uploaded to subscene😕
